frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -453,13 +503,25 @@ const setup = async ({
     setupErrorParameterValuesEndpoints();
   }
 
+  if (!showMetabaseLinks) {
+    setupEnterprisePlugins();
```

Comment:
We usually don't do that and have separate test specs that enable plugins. The reason is that the module cache is not invalidated properly and for some tests the plugins could be still enabled. Take a look at other tests that follow this structure: `common.unit.spec.tsx`, `enterprise.unit.spec.tsx`